content/productivity/crawl-planets/plugin.js

```diff
@@ -236,25 +263,33 @@ class Plugin {
         globalButton.style.marginBottom = '10px';
         globalButton.innerHTML = 'Crawl everything!'
```

Comment:
If this change goes through, I think this would be renamed something like
```suggestion
        globalButton.innerHTML = 'Crawl everything near selected!'
```